runtime/python_bridge.py

```
@@ -10,6 +10,15 @@
 import uuid
 from pathlib import Path, PurePath
 
+# Ensure the working directory is importable so local modules can be resolved when
+# the bridge is launched as a script from a different directory.
+try:
+    cwd = os.getcwd()
+    if cwd and cwd not in sys.path:
+        sys.path.insert(0, cwd)
+except Exception:
+    pass
```

Ruff (0.14.11)

`try`-`except`-`pass` detected, consider logging the exception

(S110)